proto/vschema.proto

```diff
@@ -126,6 +126,7 @@ message AutoIncrement {
 message Column {
   string name = 1;
   query.Type type = 2;
+  bool invisible = 3;
```

Comment:
I do not think we need this information in proto. possibly we directly use the proto struct so there might be not a better alternative here.

Reply:
I checked, looks like It does not need to be part of proto.

Reply:
@harshit-gangal I wonder though, do we need it in the proto for older versions? Or could we also fix it there without that?

Reply:
I think that we should be consistent and allow this setting in the proto until we deprecate the manual specifying of column information in the vschema.

Reply:
currently due to authoritative column information provided by vschema proto, we might keep this information and use it if provided. so let's keep this field till we do not deprecate this in vschema.